Index: src/main/java/com/betternpchighlight/BetterNpcHighlightPanel.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/betternpchighlight/BetterNpcHighlightPanel.java b/src/main/java/com/betternpchighlight/BetterNpcHighlightPanel.java
--- a/src/main/java/com/betternpchighlight/BetterNpcHighlightPanel.java
+++ b/src/main/java/com/betternpchighlight/BetterNpcHighlightPanel.java	(date 1755268040303)
@@ -1316,7 +1316,7 @@
                     JMenuItem setWidth = new JMenuItem("Outline width: " + (int)outlineWidth);
                     setWidth.addActionListener(ev -> {
                         int originalWidth = (int) getOutlineWidth();
-                        SpinnerNumberModel model = new SpinnerNumberModel(originalWidth, 0, 50, 1);
+                        SpinnerNumberModel model = new SpinnerNumberModel(originalWidth, 1, 50, 1);
                         JSpinner spinner = new JSpinner(model);
                         spinner.addChangeListener(changeEvent -> {
                             setOutlineWidth((Integer) spinner.getValue());
